steps/slack-step-get-member-id-by-email/README.md

```diff
@@ -0,0 +1,29 @@
+# slack-step-get-member-id-by-email
+
+This [Slack](https://slack.com) step gets a users member id by providing an email address.
+
+## Specification
+
+| Setting | Data type | Description | Default | Required |
+|---------|-----------|-------------|---------|----------|
+| `connection` | Relay Connection   | Connection to Slack requiring api token | None | True |
+| `email` | string | Channel to send message to | None | True | 
+
+## Permissions 
+
+This step requires that you have the following OAuth scopes enabled in your Slack App:
+
+* users:read
+* users:read.email
+
+## Example  
```

Comment:
Could you also document the outputs? Eg: https://github.com/relay-integrations/relay-postgres/tree/e1f524eaf3a252650f28da668f48fe062131187d/steps/query#outputs

And maybe having a shell echo or slack message step that uses it would be nice for the example.